docs/setup/native.md

```diff
@@ -72,3 +72,30 @@ the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd
 1. Create dev database: `bundle exec rake db:setup`
 1. Go to the file `config/settings/development.yml` in your local vets-api. Switch the commented out lines pertaining to the cache_dir: uncomment out line 14 (what you use for running the app via Rails), and comment out line 15 (what you use for running the app via Docker).
 1. Make sure you have the [vets-api-mockdata](https://github.com/department-of-veterans-affairs/vets-api-mockdata) repo locally installed
+
+#### Alternative (Ubuntu 18.04 LTS)
+1. Install Postgres, PostGIS, Redis, ImageMagick, Poppler, ClamAV, etc
+   - From the `vets-api` directory, run `./bin/install-ubuntu-packages`
+1. Edit `/etc/ImageMagick-6/policy.xml` and remove the lines below the comment `<!-- disable ghostscript format types -->`
```

Comment:
This may be a Ubuntu 18 specific thing, but I didn't have any issues with ImageMagick 6.9